src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/RazorFormattingEndpoint.cs

```diff
@@ -65,9 +66,17 @@ internal class RazorFormattingEndpoint : IDocumentFormattingHandler, IDocumentRa
             _logger = loggerFactory.CreateLogger<RazorFormattingEndpoint>();
         }
 
-        public TextDocumentRegistrationOptions GetRegistrationOptions()
+        DocumentRangeFormattingRegistrationOptions IRegistration<DocumentRangeFormattingRegistrationOptions>.GetRegistrationOptions()
```

Comment:
Access modifiers

Reply:
This is actually not allowed because this method is set to only show when the object is accessed as a `IRegistration<DocumentRangeFormattingRegistrationOptions>`, further modifiers are forbidden.